### DISCUSSION TITLE
Let's Grow Sunflower Land: Discussions & Ideas

### DISCUSSION THREAD

**Discussion by poteitouweb3** [sunflower-land/sunflower-land#2481] [Ideas]
### **Reaching out to Web 2**

Definitely the idea of play as guest was exceptional in bringing in players unfamiliar with blockchain, tokens and cryptos. However I believe we can go further! For this I have thought about the following proposals:

- **Content Creators**: This very fundamental role can help by creating content that shows (besides the official tutorial) strategies to use well the 3 Block Bucks.

- **Zero penalty for new players**: I don't know about you but in my opinion a farm with few or no expansion and a poorly evolved bumpkin cannot hardly cause damage to the Token's health. Many of these players wish to cash out their SFLs often to recoup investments and get more MATICs for fees. The 30% that these players will pay for withdrawal and the $0.25 to list resources in the balloon can make a huge difference in the experience of these incoming players without damaging the health of the project or being dishonest with other players. 

- **Sponsorship system**: Basically the idea is to allow the farmer who brought in a new player to help him on a one-off basis without paying anything beyond the network fee. This help could be sending resources, food, and even loaning nfts (there would be limitations on which ones can be sent). It is a way to help new players at the expense of old players "burning" resources for it. Can you imagine? A veteran player can invite and help all his friends and they have all kinds of healthy incentives to continue in the project without large investments!

- **Optional adsense for newcomers**: Obviously many of us would not like Adsenses polluting our screen, but we know the difficulties that beginners may have and this can be a viable option to fund block bucks for example. Thousands of mobile games on playstore use this mechanic and it works well!

### **Working on branding for Sunflower-land**

_"Branding is the set of actions aligned to the positioning, purpose and values of the brand. The goal of branding is to arouse sensations and create conscious and unconscious connections, which will be crucial for the customer to choose your brand at the moment of decision to buy the product or service."_

The first time I saw Sunflower-land (Still Sunflower Farms) what I got were sensations of happiness, calm, relaxation, light and healthy fun, cooperation.... And even after years the feeling is still extremely the same. That is branding!

Every person who sees the logo, name, images, videos, should be branded by it. Starting from that principle, I came up with the following ideas:

- **Clothes and Souvenir Store**: As an official and registered brand, we can create clothes, caps, mugs, pillows, cups, everything conceptual of the game! In time we would be all over the world and millions of people would come and wonder "what is Sunflower Land?"

- **Collabs**: It's definitely a fundamental idea and even quite common. Can you imagine us with a metaverse in The Sandbox? Thousands of players entering the metaverse, doing quests and getting to know the project? That would be amazing!

Finally, I would like to hear from you! Ideas, discussions, opinions. Let's start a nice thread here.

**Comment by CarrotCounter** [comment]
Thank you for opening the thread. Some really nice ideas here and the team are already working on some in the background. I'll do my best to address each of these below!!

**Content Creators:** We see this as a strong piece of increasing engagement and acquiring new players. Would love your ideas on who you would like to see producing more content for Sunflower Land. We are looking at this as we speak. 

**Zero penalty for new users:** FTUE is a big focus for the team. Our approach was to release guest mode and continually improve the experience. We agree, that for new players, the first experience needs to be highly engaging and easy enough to continue playing. We are continually working on FTUE and onboarding. 

For the 30% withdrawal tax, unfortunately, there are people in Web3 with the only priority of extracting tokens from projects, rather than playing the game. The 30% is a safeguard against these extractors. Removing this for new players would be an extremely high risk for botting. 

**Sponsorship:** I like this idea, and would say that you could already send resources to your friends to help them get started. Is there any hurdles or anything more we could do to facilitate this? 

Renting NFTs is an interesting concept, and would significantly change the utility of in-game items. Would be open to more feedback on this concept from the community, as there would be pros & cons to this approach. 

**Adsense** I personally am a fan of this idea, but cannot speak for the rest of the team and if it would be implemented. The way I see it working is that a players has 2 options to restocking their seeds: 

- watch a series of ads
- pay blockbucks

For the anti-ad players (purest), you still have the option to continue playing Sunflower Land un-interrupted by ads by using blockbucks. For players coming from Web2 used to F2P, the experience is what they are used to, and more cost effective. 

Another issue to consider under this approach is that players will still need to sync data on-chain, and players will need MATIC in their wallets to pay gas. 

Again, this is just for discussion purposes and no indication whether this approach would be taken, but would love feedback. 

**Clothes and Souvenir Store:** Team are working on this. Will hopefully be an update soon. 

**Collabs** Where there are opportunities to do collabs to increase our player base, we will investigate. We are interested in working with other NFT projects or games where we can offer Sunflower Land Utility to their token holders. 

A virtual experience in a Metaverse is a big commitment from the team right now and would need to be weighed up against the other items on our roadmap. Providing Sunflower Land utility to other NFT projects or games is more manageable in the shorter term. 

Open to suggestions here too :)

Great ideas here and open to more discussion from the community. Reminder again, this is just for discussion purposes and ideas may/ may not end up in Sunflower Land. 

Thanks for sharing :)

**Reply by poteitouweb3** [reply]
Glad to see you here @CarrotCounter ! Thanks for pointing it out!

**Comment by future2back** [comment]
About **guest players**, i like it. If all farms start out the same, why don't we let new players enter as a guest on the first starter sync (limited) to they feel the game? If player likes it, will have to mint its farm ($5 + fees) and a real Bumpkin (FREE w/ farm + fees) for it.

**Sponsorship / Renting NFT**  It can be dangerous and increase the chances of abuse.

**Adsense** It's an idea that needs to be thought about. Isn't it possible for the user to also receive the MATIC for the sync? Some system that controls the gas rate and estimates when the user will be able to sync. The value in MATIC could be stored in a smart contract before being sent to the user in time when a sync is requested.


We could partner with loopipay! Make a in-game engine to facilitate the deposits and amounts received from referrals to be invested in the budget for continuous growth campaigns of the game.

We should also implement in-game swap.
SFL > MATIC
MATIC > SFL
If users pay swap fees to third parties, why not pay them to ourselves? The users will always have the freedom of where they wants to swap, but it would be an alternative, in addition to making life easier for users not used to.

**Reply by CarrotCounter** [reply]
@future2back Thanks for your responses.

**Guest mode** is now live, and working as you suggested :)

Can you clarify what you mean around:

1. Players receiving MATIC for their sync?
2. "Make an in-game engine to facilitate the deposits and amounts received from referrals to be invested in the budget for continuous growth campaigns of the game."

Open to partnership with Loopipay. Update with on-ramping coming soon. 

There already is an in-game swap feature, where you can swap MATIC > SFL.

**Reply by future2back** [reply]
Thanks for replying @CarrotCounter !

How can i test the new feature of being guest player bro? 

Sorry, I'll try to better explain it.


1 - On Adsense mode, instead of player receive blockbucks, receive MATIC, not exactly MATIC , but a sync on chain.

2 - Loopipay pays a bonus to the referred user, so if we have a in game engine to facilitate the deposits, that I didn't have to leave the game to do, using our ref in Loopipay we could get a budget to continuous investing on our growth campaigns. This is just an idea, if you get in touch with Loopipay sure will get a better partnership.


Add new ideas:

- Could we add a NPC on Goblin Retreat that works with niftyswap engine?
- I see a lot of our users spending their time on third party websites. Statistics site and to have information that we don't provide. Why not keep our users with us? Why not add these tools to our interface? We have to value keeping the user time on our website.
- Could we work on a news maps that requires the user leave their farm, failing to generate new resources? Stone Haven and Snow Kingdom can be used for this?
- Don't you think that our interface can be confusing at some points? 
"How to play ?" button is on Engine > ... > How to play ? (Btw its not updated) 
"Refresh" button is so important, lot ot old players dont know this button and its functionality... its inside of Engine > ... > Settings > Refresh
"Add SFL / Add matic"  Why not adding it at Bank on Goblin Retreat?







Thank you.

**Reply by CarrotCounter** [reply]
I got you..

With guest mode, it detects if you have a farm already. It will not provide the option to enter guest mode if a farm is detected. For your knowledge, it is quite basic and will be improved upon over time. Might be best to access through a new player. 

**Adsense** - understand what you say here... but i don't think it would make sense.  Earning MATIC for watching ads in Sunflower Land would require the project to purchase MATIC from ad revenue to then provide to the player - hence would be eliminating one of the projects core revenue. Also, i think giving players MATIC for watching ads could be quite dangerous and attract the wrong people. 

We will be expanding on the Block Bucks utility and to me this is where it would make sense. 

The issue you have highlighted though is the MATIC sync... We are constantly improving this as it is not great for user experience. 

**LoopiPay** - more to come on LoopiPay particularly as we are working with Poko, an on-ramp aggregator. 

**NiftySwap** - for now, this is not in-game as it would cannibalise the trading feature between players. 

Other suggestions noted, and agree. It's a matter of priority with our small team, and if we would prioritise over the amazing other features on our roadmap. 

Thanks for your feedback :)

**Reply by future2back** [reply]
I saw guest mode on youtube but tried it with different account here but didnt work... Is guest mode working with Metamask?


Thank you.

**Reply by guilhermedecampo** [reply]
Maybe you can try https://rampbr.com also ;) We are thinking in APIs too 👀
